README.md

```diff
@@ -1,4 +1,5 @@
 ![pytests](https://github.com/google/gcp_scanner/actions/workflows/python-app.yml/badge.svg)
+[![GCP scanner version](https://img.shields.io/badge/version-1.0.0-blue.svg)](https://github.com/google/gcp_scanner/releases/tag/v1.0.0)
```

Comment:
Do we need to update it manually each time we make new release?

Reply:
> Do we need to update it manually each time we make new release?

Yes, we need to update the version number in the badge manually each time we make a new release. or we can make automate the process of updating the version tag badges by using Shields.io, Travis CI, or CircleCI tools which are all third-party tools and services.

Reply:
I think it is not very scalable. I am pretty sure there should be some solution via `latest` or etc. Similar to other projects https://github.com/google/guava/edit/master/README.md

Reply:
> I think it is not very scalable. I am pretty sure there should be some solution via `latest` or etc. Similar to other projects https://github.com/google/guava/edit/master/README.md

The guava project has used Shields.io for automating it. Can I use it here?

Reply:
> I think it is not very scalable. I am pretty sure there should be some solution via `latest` or etc. Similar to other projects https://github.com/google/guava/edit/master/README.md

Ok, got it. I have modified it, now it will automatically change its version

Reply:
@mshudrak I have submitted the proposal, How can I add this PR to it?

Reply:
Hm, not sure. We will check for PRs made by applicants anyway to have a fresh list.

Reply:
Ok, got it. Thank you